text/0000-contributor-team.md

```diff
@@ -0,0 +1,39 @@
+- Start Date: 2019-02-27
+- RFC PR:
+- unified Issue:
+
+# Summary
+
+It'd be nice to add a "contributor" team to unified organizations which
+all contributors are invited to. There's precedent here with what Gatsby
+does and helps contributors receive a warm welcome to the community after
+getting a PR merged.
+
+## Basic example
+
+If a user gets a PR merged they receive an invite to the org as a "contributor".
+The contributor team doesn't have write access.
+
+## Motivation
+
+To make contributors feel welcome.
+
+## Detailed design
+
+For now we can manually handle it, but eventually we can use a GitHub Action.
+
+## Drawbacks
+
+None that I can see.
+
+## Alternatives
+
+N/A
+
+## Adoption strategy
+
+N/A
+
+## Unresolved questions
+
+N/A
```

Comment:
What (if any) access do read-only members have to GitHub integrated platforms? (Zeit Now, Travis CI, etc)